library/newrelic-usage/dashboards/nr_usage_apof.json

```diff
@@ -0,0 +1,350 @@
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 1,
+            "row": 19,
+            "height": 3,
+            "width": 4
+          },
+          "title": "How many GBs have I accrued this month?",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrConsumption select sum(GigabytesIngested) as 'Total GBs Ingested this Month' where productLine = 'DataPlatform' SINCE this month"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 5,
+            "row": 19,
+            "height": 3,
+            "width": 4
+          },
+          "title": "How many GBs have I accrued since my contract start date?",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrConsumption select sum(GigabytesIngested) as 'Total GBs Ingested Since Start Date' where productLine = 'DataPlatform' SINCE '2020-09-12'"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 9,
+            "row": 19,
+            "height": 3,
+            "width": 4
+          },
+          "title": "How many GBs have been ingested by month?",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM NrConsumption SELECT sum(GigabytesIngested) AS 'Gigabytes Ingested' WHERE productLine ='DataPlatform' SINCE '2020-09-12' FACET monthOf(timestamp)"
```

Comment:
It is meant to reflect the contract start date for the customer. Yes, we can either remove it, change it to 'this year' to reflect since the beginning of the year or change the widget title to indicate that the user must update the query with their contract start date. I'm open to suggestions. What's the best practice here?
```suggestion
                "query": "FROM NrConsumption SELECT sum(GigabytesIngested) AS 'Gigabytes Ingested' WHERE productLine ='DataPlatform' SINCE this year FACET monthOf(timestamp)"
```